### PR TITLE
support values with equals in them

### PR DESCRIPTION
**Pull request by alanclarke**
@alexjesp @roberttod

### CODE REVIEW

**Comment by alexjesp** [comment]
:+1:

**Comment by roberttod** [comment]
REBASE

**Comment by roberttod** [comment]
Also, don't use semicolons

File: test/cookieman_test.js [hunk 1]
```diff
@@ -56,12 +56,14 @@ define(function (require) {
...
         document.cookie = "flippy=magoo1;";
+        document.cookie = "foo=ba=r;";
       });
 
       afterEach(function () {
         document.cookie = "sweety=darling; expires=" + new Date(1).toUTCString();
-        document.cookie = "flippy=magoo; expires=" + new Date(1).toUTCString();
         document.cookie = "flippy=magoo; path=/cookie; expires=" + new Date(1).toUTCString();
+        document.cookie = "flippy=magoo1; expires=" + new Date(1).toUTCString();
+        document.cookie = "foo=ba=r; expires=" + new Date(1).toUTCString();
```
Comment (roberttod) [inline test/cookieman_test.js]:
```javascript
document.cookie = "foo=barn=; expires=" + new Date(1).toUTCString();
document.cookie = "foo==barn=; expires=" + new Date(1).toUTCString();
```

**Comment by roberttod** [comment]
:fallen_leaf:

**Comment by roberttod** [comment]
Leave it, ship it

**Comment by roberttod** [comment]
*Leaf it alone now

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+pids
+logs
+results
+
+npm-debug.log
+node_modules
+
+
+#bower
+bower_components
```
Comment (KidkArolis) [inline .gitignore]: whyyy

Comment (KidkArolis) [inline .gitignore]: noooo

Comment (roberttod) [inline .gitignore]: ^ lol

**Pull request merged**